### DISCUSSION TITLE
Question - Are you considering Rome Frontend toolchain to replace Babel, ESLint, webpack, Prettier, Jest, etc?

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/next.js#16754] [Help]
Article: https://www.infoq.com/news/2020/03/rome-experimental-js-toolchain/
Github: https://github.com/romefrontend/rome

This tool (Rome) seems to be of very high quality and aims at replacing the whole JS toolchain (linters, etc.) and combine many tools into one.

I know it's a bit early, but the project has drawn quite a bit of attention, with 13k stars already.

I'd like to know if you're considering it to eventually replace some tools currently used by Next.js (e.g: babel, webpack, eslint)

Has there been any attempts/tests/POCs about replacing the current toolchain with Rome?

Not only am I interested to know about whether it'll integrate with Next.js (and whether it'll be compatible with the Next.js/Vercel build chain, instead of webpack), but I'm also interested about using it within [Next Right Now](https://github.com/UnlyEd/next-right-now) for all the benefits it might bring.

At the time of writing, Rome can only be used as a [JS/TS Linter](https://romefrontend.dev/#development-status).
![image](https://user-images.githubusercontent.com/3807458/91878530-bcc12e00-ec7f-11ea-9afa-f07a0c3aeab8.png)

**Comment by sethsnel** [comment]
Any updates on this inquiry?

**Reply by Vadorequest** [reply]
I haven't dug deeper so far. To my understanding, it's still under work.

I haven't checked since then, here is the status page:
https://rome.tools/#development-status

It doesn't seem to have changed much since last time:
![image](https://user-images.githubusercontent.com/3807458/99704267-6ef6dd80-2a98-11eb-8770-a17386a68a3f.png)

It's still in the "Linter" step, and cannot be used as a full replacement for all the tools it aims to be a replacement for, at this time.

**Reply by sethsnel** [reply]
Hmm it seemed promising, hopefully development isn't stagnated.

**Comment by pioz** [comment]
Any updates on this? @ijjk @timneutkens

**Comment by tgmarinho** [comment]
??

**Comment by GabenGar** [comment]
As per [the site](https://rome.tools) it will only get css support at the end of the 2023, so no point nagging the devs until then.
